cli/schema/cypress.schema.json

```
@@ -45,7 +45,7 @@
     "testFiles": {
       "type": "string",
       "default": "**/*.*",
-      "description": "A String glob pattern of the test files to load"
+      "description": "A String or Array of string glob pattern of the test files to load. See https://on.cypress.io/configuration#Global"
```

doesn't sound right as is, maybe something like
```suggestion
      "description": "A String or Array of glob patterns of test files to load. See https://on.cypress.io/configuration#Global"
```

---

Agreed and changed.